test/WebJobs.Extensions.RabbitMQ.Samples/RabbitMQSamples.cs

```diff
@@ -76,7 +70,7 @@ public static class RabbitMQSamples
 
         // Trigger samples
         public static void RabbitMQTrigger_String(
-             [RabbitMQTrigger("localhost", "queue")] string message,
+             [RabbitMQTrigger("amqp://guest:guest@localhost:5672/", "queue")] string message,

```

Comment:
Add test that uses username and password. Note: UserName and password should be setting names as well similar to Twilio SID and password